cfme/common/candu_views.py

```diff
@@ -37,3 +27,32 @@ def is_displayed(self):
             )
         else:
             return "Capacity & Utilization data for Virtual Machine" in self.title.text
+
+
+class VirtualcenterVMUtilizationView(VMUtilizationView):
```

Comment:
**REQUIRED**: This is a no-no I'm afraid. It's a real shame that these are not linking the integer with the particular graph. However, what we need to do, is move these Views to the provider modules. They are too specific to be here.